rplugin/python3/deoplete/sources/deoplete_jedi.py

```diff
@@ -33,34 +34,46 @@ def on_init(self, context):
         vars = context['vars']
 
         self.statement_length = vars.get(
-            'deoplete#sources#jedi#statement_length', 0
-        )
+            'deoplete#sources#jedi#statement_length', 0)
         self.use_short_types = vars.get(
-            'deoplete#sources#jedi#short_types', False
-        )
+            'deoplete#sources#jedi#short_types', False)
         self.show_docstring = vars.get(
-            'deoplete#sources#jedi#show_docstring', False
-        )
+            'deoplete#sources#jedi#show_docstring', False)
+        self.debug_server = vars.get(
+            'deoplete#sources#jedi#debug_server', False)
```

Comment:
With only `g:deoplete#sources#jedi#debug_server = 1` debugging should be enabled already - assuming an implicit call to `call deoplete#custom#set('jedi', 'debug_enabled', 1)`.